.github/workflows/cloudflare.yml

```diff
@@ -57,7 +66,7 @@ jobs:
       - name: Yarn Install
         run: yarn install --immutable
       - name: Build Web
-        run: yarn env && yarn build:web
+        run: MODE=${{ env.BUILD_MODE }} DISABLE_SOURCE_MAP=true yarn build:web 
```

Comment:
```suggestion
        run: MODE=${{ env.BUILD_MODE }} DISABLE_SOURCE_MAP=true yarn build:web
```

Reply:
Removed in [780fc35](https://github.com/shapeshift/web/pull/8924/commits/780fc358dd698fa1d4f35579acc5f620dc5da98a)